demo/package.json

```diff
@@ -12,13 +12,10 @@
     "test": "cross-env NODE_ENV=test jest"
   },
   "dependencies": {
-    "@labkey/api": "1.1.1",
-    "@labkey/components": "1.1.1",
-    "react": "16.13.1",
-    "react-dom": "16.13.1"
+    "@labkey/components": "1.9.0"
   },
   "devDependencies": {
-    "@labkey/build": "0.4.0",
+    "@labkey/build": "0.5.0",
```

Comment:
Seems like we should go ahead and make a 1.0.0 version of @labkey/build whenever we make our next changes since it is pretty stable at this point.

Reply:
Yes, good point.